tests/test_lab_evaluate.py

```diff
@@ -540,3 +543,26 @@ def test_invalid_model_path_mt_bench(cli_runner: CliRunner, tmp_path):
         in result.output
     )
     assert result.exit_code != 0
+
+
+def test_vllm_args_None(cli_runner: CliRunner):
```

Comment:
Is this a common pattern to use case like this?  Hurts my eyes but might just be me.

Reply:
I was trying to reflect the `None` name, but I can change to `null` which is how it's represented in file.